### ISSUE TITLE
Cave Air doesn't light up

### ISSUE THREAD

**Issue by Venaf** [Tschipcraft/dynamiclights#2]
### Describe the bug

Working great so far, but while caving certain places don't work - started noticing it in mineshafts.  I don't know how prevalent cave_air is with the new cave generation systems, but that seems to be where it fails - if I step into an area I dug out myself, it works fine.
I did look at the datapack and you have air and cave_air tagged, but in find_target.mcfunction, maybe the 'replace air' commands don't work without cave air being specified in some way, too? 

### Expected behavior

Lights continue following you as you move through caves.

**Comment by Tschipcraft** [comment]
Yes, that's probably the cause of it. Thank you for the bug report, will be fixed in the next release!
